src/foam/core/Unit.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.core',
+  name: 'Unit',
```

Comment:
Related to another comment I made: `Unit` is supposed to have a `format :: Long -> String` method. Shouldn't that be added on this model?

Reply:
Yes it will be by @KateMengjunXia's formatting Pr